### DISCUSSION TITLE
Can't install?

### DISCUSSION THREAD

**Discussion by garrett85** [phalcon/phalcon#252] [Q&A] [answered]
Linux Mint 20.1 & PHP 7.4.3

I'm following the installation instructions at https://phalcon.io/en-us/download/linux
But his what I'm getting when I get to the following commands.

sudo apt-get install php7-phalcon
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package php7-phalcon

sudo apt-get install php7.0-phalcon
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package php7.0-phalcon
E: Couldn't find any package by glob 'php7.0-phalcon'
E: Couldn't find any package by regex 'php7.0-phalcon'

Where am I doing wrong here? Thanks.

**Comment by garrett85** [comment, marked as answer]
For anyone else who find this very unhelpful community, I was able to get this installed after running `sudo apt install php-psr`

**Reply by niden** [reply]
![image](https://user-images.githubusercontent.com/1073784/171201984-9d6568e8-c5ac-436f-963c-fee936359ff3.png)

For the version you are trying to install PSR is a requirement.
